bindings/README.md

```diff
@@ -0,0 +1,3 @@
+### Bindings
+
+Provides golang bindings for calling our [smart contracts](https://github.com/mysteriumnetwork/payments-smart-contracts) and others that might be useful like uniswap v3.
```

Comment:
It also provides bindings for these:
* https://github.com/mysteriumnetwork/rewarder-smart-contracts
* https://github.com/mysteriumnetwork/topperupper-smart-contracts